scripts/generate-json-schema.ts

```
@@ -210,42 +220,65 @@
+			Result.inspectError((error) => {
+				logger.error(`Failed to write ${SCHEMA_FILENAME}:`, error);
+				process.exit(1);
+			}),
+			Result.inspect(() => {
+				logger.info(`✓ Generated ${SCHEMA_FILENAME}`);
+				filesToLint.push(SCHEMA_FILENAME);
+			}),
+		);
+	}
+
+	if (docsSchemaChanged) {
```

ESLint

[error] Unexpected any value in conditional. An explicit comparison or type conversion is required.

(ts/strict-boolean-expressions)